dist/images/ovnkube.sh

```diff
@@ -987,6 +987,11 @@ local-nb-ovsdb() {
   wait_for_event attempts=3 process_ready ovnnb_db
   echo "=============== nb-ovsdb (unix sockets only) ========== RUNNING"
 
+ [[ "local" == "${OVN_GATEWAY_MODE}" &&  "true" == "${OVN_ROUTE_ADVERTISEMENTS_ENABLE}" ]] && {
+    ovn-nbctl set NB_Global . options:use_ct_inv_match=false
+    echo "=============== nb-ovsdb ========== reconfigured for route advertisements"
+  }
```

Comment:
what will this look like in OpenShift ? I assume CNO will have to set something similar on the OVS nodes of the cluster. 

I do wonder if it makes sense to expose this global OVS table option via nmstate,  and have it configurable via NNCP (nmstate policy).

Reply:
There will be a CNO change yeah, this is not really something that matters for upstream though.
Here is the pr: https://github.com/openshift/cluster-network-operator/pull/2735